src/PermissionResourceTrait.php

```diff
@@ -16,6 +16,15 @@
 
 trait PermissionResourceTrait
 {
+
+	/**
+	 * Override the applyFilters method,title field translation
+	 */
+	public function title() {
+
```

Comment:
By default, display_names is not defined. This is a breaking change. Please exclude this from the PR